jcli/src/jcli_lib/transaction/staging.rs

```diff
@@ -95,6 +98,16 @@ impl Staging {
         })
     }
 
+    pub fn set_validity_time(&mut self, input: interfaces::BlockDate) -> Result<(), Error> {
```

Comment:
Name `set_valid_until` for consistency? Or better, `set_expiry_date` everywhere because that's the name used in the library API.